pkgs/servers/monitoring/prometheus/default.nix

```diff
@@ -30,131 +28,85 @@
 }:
 
 let
-  version = "2.30.3";
+  version = "2.35.0";
 
   src = fetchFromGitHub {
```

Comment:
Should be possible to move this into `buildGoModule` as it's no longer needed by other variables.